src/overlays/actors/ovl_En_Wood02/z_en_wood02.c

```diff
@@ -15,9 +15,40 @@ void EnWood02_Destroy(Actor* thisx, GlobalContext* globalCtx);
 void EnWood02_Update(Actor* thisx, GlobalContext* globalCtx);
 void EnWood02_Draw(Actor* thisx, GlobalContext* globalCtx);
 
-extern UNK_TYPE D_06000700;
+typedef enum {
+    /* 0x00 */ TREE_CONICAL_LARGE,
+    /* 0x01 */ TREE_CONICAL_MEDIUM,
+    /* 0x02 */ TREE_CONICAL_SMALL,
+    /* 0x03 */ TREE_CONICAL_SPAWNER,
+    /* 0x04 */ TREE_CONICAL_SPAWNED, // Improve names of these?
+    /* 0x05 */ TREE_OVAL_GREEN,
+    /* 0x06 */ TREE_OVAL_YELLOW_SPAWNER,
+    /* 0x07 */ TREE_OVAL_YELLOW_SPAWNED,
+    /* 0x08 */ TREE_OVAL_GREEN_SPAWNER,
+    /* 0x09 */ TREE_OVAL_GREEN_SPAWNED,
+    /* 0x0A */ TREE_KAKARIKO_ADULT,
+    /* 0x0B */ BUSH_GREEN_SMALL,
+    /* 0x0C */ BUSH_GREEN_LARGE,
+    /* 0x0D */ BUSH_GREEN_SMALL_SPAWNER,
+    /* 0x0E */ BUSH_GREEN_SMALL_SPAWNED,
+    /* 0x0F */ BUSH_GREEN_LARGE_SPAWNER,
+    /* 0x10 */ BUSH_GREEN_LARGE_SPAWNED,
+    /* 0x11 */ BUSH_BLACK_SMALL,
+    /* 0x12 */ BUSH_BLACK_LARGE,
+    /* 0x13 */ BUSH_BLACK_SMALL_SPAWNER,
+    /* 0x14 */ BUSH_BLACK_SMALL_SPAWNED,
+    /* 0x15 */ BUSH_BLACK_LARGE_SPAWNER,
+    /* 0x16 */ BUSH_BLACK_LARGE_SPAWNED,
+    /* 0x17 */ LEAF_GREEN,
+    /* 0x18 */ LEAF_YELLOW
+} WoodType;
```

Comment:
ideally everything in here would have the same prefix

Reply:
Okay, although it makes the names even longer. Should I also move these to the header?